Dockerfile

```diff
@@ -1,6 +1,16 @@
 FROM rethinkdb:2.3
 
-RUN apt-get update
+# From http://stackoverflow.com/a/38553499
+
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y locales
```

Comment:
@josephwinston  Why did you do `apt-get update` and `DEBIAN_FRONTEND=noninteractive apt-get install -y locales` in the same line? It's not "wrong," I'm just curious if there's some reason that I don't grok.